man/he.1

```diff
@@ -35,6 +35,8 @@ Enable the use of named character references (like `&copy;`) in the output. If c
 Encode every symbol in the input string, even safe printable ASCII symbols.
 .It Sy "--encode --allow-unsafe"
 Encode non-ASCII characters only. This leaves unsafe HTML/XML symbols like `&`, `<`, `>`, `"`, and `'` intact.
+.It Sy "--encode --decimal-output"
+Take decimal format rather than hexadecimal format for encoded HTML entities (`©` is turned into `&#169;` instead of `&#xA9;`).
```

Comment:
Let’s reword it as follows:

```
Use the decimal format rather than the hexadecimal format for encoded HTML entities, e.g. `©` is turned into `&#169;` instead of `&#xA9;`.
```